cmd/stack.go

```diff
@@ -332,3 +350,20 @@ func printStatus(cmd *cobra.Command, servicesStatus []stack.ServiceStatus) {
 	t.SetStyle(table.StyleRounded)
 	cmd.Println(t.Render())
 }
+
+func detectShell() (string, error) {
+	ppid := os.Getppid()
+
+	parent, err := sysinfo.Process(ppid)
+	if err != nil {
+		return "", fmt.Errorf("cannot retrieve information for process %d: %w", ppid, err)
+	}
+
+	parentInfo, err := parent.Info()
+	if err != nil {
+		return "", fmt.Errorf("cannot retrieve information for parent of process %d: %w", ppid, err)
+	}
+
+	shell := filepath.Base(parentInfo.Exe)
```

Comment:
In bash in Windows this detects `bash.exe`, and fails to find the shell type.

FTR, in Windows terminal it detects `cmd.exe`, and in PowerShell, `powershell.exe`. No need to add support for these now though.

Reply:
I added a trim suffix to remove any `.exe` suffix.